src/dataflow/src/render/upsert.rs

```diff
@@ -147,14 +148,14 @@ where
                                 if offset < new_offset {
                                     *value = SourceData {
```

Comment:
I think if we called the entry in the hashtable `entry` instead of `value` then we could call the `SourceOutput` value `value` instead of `val` and not having `val` and `value` in this block would make the cognitive load a bit easier for me but this is a minor nit so tioli

Reply:
Changed `entry` to `value`. Thanks!

Also changed `position` to `new_position` to make it clearer which is the new offset vs old one.

Any reduction in mental load will reduce the chance of such a bug happening again and being undetected for months.